### DISCUSSION TITLE
Polling frequency of the modbus fixed at 30sec

### DISCUSSION THREAD

**Discussion by Fernando-Vaz** [wills106/homeassistant-solax-modbus#408] [Q&A]
Anyone confirm that?
I tried other value, less than 30sec, but polling frequency is 30sec.
Thanks

**Comment by formanv75** [comment]
yeah, I have the same issue. It was 10 seconds and it changed to 30 (ignoring the setting) after update

**Reply by wills106** [reply]
@Fernando-Vaz Was this a fresh install or during a HA / Integration upgrade?

@formanv75 Did this happen when you upgraded to Home Assistant to 2023.06.0? #434 I can't workout what went wrong as I know someone who went from HA 2023.05.x to 2023.06.1 (Skipping 2023.06.0) without an issue.

**Reply by formanv75** [reply]
It was an update/upgrade to version 6.4 and I definitely skipped few previous updates. I updated from last 5.x (or release before) directly to 6.4.
Everything else is working well, just the polling time is locked to 30 seconds

and BTW I have 3 phase hybrid SolaX gen4 with the Wifi dongle.

**Reply by formanv** [reply]
I have tried few more things.
- upgrade to 6.5 - no difference
- rollback to 6.1 - it was not possible to go to 5.x - no differrence
- back to 6.5 & set the poll time to 60 seconds - it stoped reading the values 
- set the time to 10 seconds (on 6.5) - it is polling the values in 30 seconds intervals

**Reply by wills106** [reply]
Instead of upgrading / downgrading the Integration.
Have you tried simply removing the Integration from the Integration page and adding it back in?

That's all I had to do to get it working again when I updated Home Assistant from 2023.06.0b4 to release 2023.06.0
I didn't change the Integration version.

**Reply by formanv** [reply]
that helped. 
I deleted the inverter from devices and I deleted the integration from HACS.
After that I installed and configured it again.

Thanks
